helm/nessie/values.yaml

```diff
@@ -141,6 +141,27 @@ advancedConfig:
 #      console.format: "%d{HH:mm:ss} %s%e%n"
 #      category."org.projectnessie".level: DEBUG
 
+# Reverse Proxy Settings
```

Comment:
It's not clear whether this is a new section, or the continuation of the `advancedConfig` section above.